Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -480,8 +506,12 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
      * 136 (CappedPositionLost), or 237 (CursorKilled) while executing a getMore command.
      */
     func testChangeStreamDoesntResume() throws {
-        guard MongoSwiftTestCase.topologyType != .single else {
-            print(unsupportedTopologyMessage(testName: self.name))
+        let testRequirements = TestRequirement(
+            acceptableTopologies: ["replicaSet", "sharded"].map { TestTopologyConfiguration(from: $0) })
```

Comment:
ditto
This applies to a few other places in this file, so I'll stop commenting on every one.

Reply:
done